creator-node/src/logging.js

```diff
@@ -3,13 +3,23 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+function StdOutWithLevelName() {}
```

Comment:
could you add a comment that this follows the pattern here https://github.com/trentm/node-bunyan#stream-errors

Reply:
Thanks for catching this! I'll make a conscious effort to link to borrowed code in the future!

Fwiw, the borrowed code was from a community member since the author does not support this behavior natively, is hoping to add it into Bunyan 2.0 (not currently slated for release), and provided a less-than-straightforward gist.